index.js

```diff
@@ -37,4 +47,4 @@ const stringify = (object, type) => {
     }
 }
 
-module.exports = { parse, stringify }
+module.exports = { parse, stringify, removeFileExt }
```

Comment:
That 1-line function doesn't need to be tested directly I guess. In `test/cli/convert.js` there's a test for all the different types (`json`, `yaml`, etc.), you can just add all the `.`-prefixed types there (`.json`, `.yaml`, etc.). I think that's good enough:

```js
// in test/cli/convert.js line 41

const types = ['json', 'json5', 'hjson', 'cson', 'yaml', 'toml', 'ini']
const extensions = types.concat(types.map(type => '.' + type))
```